packages/api-tests/test/browser-test-setup.js

```diff
@@ -0,0 +1,70 @@
+const webdriverio = require('webdriverio');
+const selenium = require('selenium-standalone');
+
```

Comment:
Yeah they should work the same as the others. Ill make sure they run fine on my CI before merging